DEVDOCS-4711: [update] Handlebars reference, add strReplace helper (#1726)
bigcommerce/dev-docs: docs/stencil-docs/reference-docs/handlebars-helpers-reference.md

| [replace](#replace) | string | Replaces all instances of the first parameter in the second parameter. |
| [setURLQueryParam](#seturlqueryparam) | string | Appends keys values to a URL. |
| [stripQuerystring](#stripquerystring) | string | Removes a query string. |

> | [strReplace](#strreplace) | string | Replaces some or all occurrences of substring within the given string. |

Reviewer: ```suggestion
| [strReplace](#strreplace) | string | Replaces some or all occurrences of a substring within the given string. |

```

> | [strReplace](#strreplace) | string | Replaces some or all occurrences of substring within the given string. |

Author: alternatively:

```suggestion
| [strReplace](#strreplace) | string | Replaces some or all occurrences of a target substring within the subject string. |

```

> | [strReplace](#strreplace) | string | Replaces some or all occurrences of a target substring within the subject string. |

Author: ```suggestion
| [strReplace](#strreplace) | string | Replaces some or all occurrences of a target substring within a subject string. |

```

- [See it in GitHub](https://github.com/bigcommerce/paper-handlebars/blob/master/helpers/stripQuerystring.js)

### {{strReplace}}

```handlebars

> {{strReplace string substr newSubstr occurrence}}

Author: ```suggestion
{{strReplace subjectString targetSubstring newSubstring occurrenceSelection}}

```

```

> Replaces some or all occurrences of substring within the given string.

Reviewer: ```suggestion
Replaces some or all occurrences of a substring within the given string.

```

> Replaces some or all occurrences of substring within the given string.

Author: alternatively:

```suggestion
Replaces some or all occurrences of the target substring within the subject string.

```

#### Parameters

> - `string` {String}: The original string.

Author: ```suggestion
- `subjectString` {String}: The original string to modify.

```

> - `substr` {String}: The substring in `string` to be replaced.

Author: ```suggestion
- `targetSubstring` {String}: The substring in `string` to be replaced.

```

> - `newSubstr` {String}: The substring to replace `substr`.

Author: ```suggestion
- `newSubstring` {String}: The substring you want to insert in place of `targetSubstring`.

```

> - `occurence` {Number}: The number of occurrences of `substr` to be replaced by `newSubstr` in `string`.

Author: ```suggestion
- `occurrenceSelection` {Integer}: Optional. The number of occurrences of `targetSubstring` to replace with `newSubstring`. If you pass no argument or the supplied integer is greater than the number of `targetSubstring` occurrences, `newSubstring` will replace all instances of `targetSubstring`. A positive integer will replace `targetSubstring` starting from the beginning of the line;  a negative integer will replace from the end of the line.

```

Author: test end of line condition and modify this description or line 1251-1252 accordinly.

Author: suggestion updated following collaboration.
